src/Bicep.Core.Samples/DataSet.cs

```diff
@@ -28,6 +28,7 @@ public class DataSet
         public const string TestFileMainParamSyntax = "main.syntax.bicepparam";
         public const string TestFileMainFormatted = "main.formatted.bicep";
         public const string TestFileMainCompiled = "main.json";
+        public const string TestFileMainCompiledParm = "main.bicepparam.json";
```

Comment:
To avoid confusion, I suggest renaming the input file to `parameters.bicepparam` instead of `main.bicepparam` - that way your output file will just be named `parameters.json`

Reply:
Good idea, will do that